.spelling

```diff
@@ -269,6 +269,7 @@ iptables
 IPv4
 IPv6
 Istio
+istioctl
```

Comment:
commands should be in backticks in the documentatiton (such as istioctl or kubectl).  Please remove istioctl from spelling;

Reply:
addressed